### ISSUE TITLE
remove markdown before checking if a message is a haiku

### ISSUE THREAD

**Issue by twentylemon** [duck-dynasty/duckbot#669]
Title. We currently remove a hardcoded list of symbols from the message, but we should also clean up markdown as well.

Discord.py's [remove_markdown](https://discordpy.readthedocs.io/en/stable/api.html#discord.utils.remove_markdown) might be useful.

* https://github.com/duck-dynasty/duckbot/blob/main/duckbot/cogs/messages/haiku.py#L35

**Comment by jameshughes89** [comment]
Mildly-ish related to #525

**Comment by twentylemon** [comment]
Yes and no. Though we remove punctuation when checking if a message is a haiku, we keep that punctuation when producing the resultant message. So like `three, two seven, two three` (words being the number of syllables they are would give:
> three, two
> seven,
> two three

We essentially remove the punctuation so we can lookup how many syllables a word is. So a message like `three **two** _seven two_ three` should yield:
> three **two**
> _seven_
> _two_ three
